src/components/message-input/index.tsx

```diff
@@ -1,32 +1,34 @@
 import React, { RefObject } from 'react';
 import { MentionsInput, Mention } from 'react-mentions';
 import Dropzone from 'react-dropzone';
-import classNames from 'classnames';
-import { emojiMentionsConfig, userMentionsConfig } from './mentions-config';
+
+import { config } from '../../config';
 import { Key } from '../../lib/keyboard-search';
+import { MediaType } from '../../store/messages';
+import { emojiMentionsConfig, userMentionsConfig } from './mentions-config';
 import { UserForMention, Media, dropzoneToMedia, addImagePreview, windowClipboard } from './utils';
-import Menu from './menu';
-import ImageCards from '../../platform-apps/channels/image-cards';
-import { config } from '../../config';
+
+import Menu from './menu/menu';
+import { IconButton } from '../icon-button';
+import { EmojiPicker } from './emoji-picker/emoji-picker';
 import ReplyCard from '../reply-card/reply-card';
-import { IconFaceSmile, IconStickerCircle } from '@zero-tech/zui/icons';
+import MessageAudioRecorder from '../message-audio-recorder';
+import { Giphy, Properties as GiphyProperties } from './giphy/giphy';
 import { ViewModes } from '../../shared-components/theme-engine';
-import { PublicProperties as PublicPropertiesContainer } from './container';
-import { EmojiPicker } from './emoji-picker';
-import { IconButton } from '../icon-button';
-import { IconMicrophone2 } from '@zero-tech/zui/icons';
 import AudioCards from '../../platform-apps/channels/audio-cards';
-import MessageAudioRecorder from '../message-audio-recorder';
-import { Giphy, Properties as GiphyProperties } from './giphy';
+import ImageCards from '../../platform-apps/channels/image-cards';
+import AttachmentCards from '../../platform-apps/channels/attachment-cards';
+import { PublicProperties as PublicPropertiesContainer } from './container';
+import { IconFaceSmile, IconSend3, IconMicrophone2, IconStickerCircle } from '@zero-tech/zui/icons';
 
+import classNames from 'classnames';
 import './styles.scss';
-import { MediaType } from '../../store/messages';
-import AttachmentCards from '../../platform-apps/channels/attachment-cards';
 
 export interface PublicProperties extends PublicPropertiesContainer {}
 
 export interface Properties extends PublicPropertiesContainer {
   viewMode: ViewModes;
+  isMessengerFullScreen?: boolean;
```

Comment:
I see that we're using `isFullScreen` above for the same. Maybe use that here as well (if possible)?

Reply:
I think for the `message-input`, `isMessengerFullscreen` gives it more context of what the attribute is for. Rather than assuming what exactly is `isFullScreen`